Cargo.lock

```diff
@@ -304,7 +356,7 @@ version = "0.0.10"
 dependencies = [
  "anyhow",
  "chrono",
- "crossterm",
+ "crossterm 0.19.0",
```

Comment:
We seem to be pulling two crossterm versions. You might want to update the version specified in our Cargo.tomls

Reply:
Ah, you updated -view but not -term